rust/src/libnm_dbus/connection/bridge.rs

```diff
@@ -58,19 +58,20 @@ impl NmSettingBridge {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Default)]
+#[derive(Debug, Clone, PartialEq, Default, Deserialize)]
+#[serde(try_from = "DbusDictionary")]
 pub struct NmSettingBridgePort {
     _other: HashMap<String, zvariant::OwnedValue>,
 }
 
-impl TryFrom<HashMap<String, zvariant::OwnedValue>> for NmSettingBridgePort {
+impl TryFrom<DbusDictionary> for NmSettingBridgePort {
     type Error = NmError;
     fn try_from(
         setting_value: HashMap<String, zvariant::OwnedValue>,
```

Comment:
The TryFrom is using `DbusDictionary`, so the `fn try_from` should be also using `DbusDictionary`

Reply:
Good catch, thanks, done